doc/api.txt

```diff
@@ -376,6 +376,7 @@ mechanism and key "00000000000000000000000000000000".
                     "key encipherment",
                     "server auth"
                 ]
+                "name_whitelist": "^.*cloudflare.com",
```

Comment:
I think this should be

```
                 "name_whitelist": `^[\w_-]*\.cloudflare.com$`, 
```

(See [this](http://play.golang.org/p/LkmhghLKju) and [this](http://play.golang.org/p/vx5wWXBcBV).)